src/shared/ui/SearchForm/SearchForm.js

```diff
@@ -12,20 +12,20 @@ export const SearchForm = ({ initialValue = "", onSearch }) => {
     onSearch(searchText);
   };
 
-  const handleKeyPress=(event)=>{
+  const handleKeyPress = (event) => {
     if (event.key === "Enter") {
       onSearch(searchText);
     }
   }
 
   return (
-    <form class="search-form input-element">
-      <input type="text" placeholder="Search...."
+    <form className="search-form input-element">
```

Comment:
you don't have any actions for submitting 
```suggestion
    <form className="search-form input-element" onSubmit={()=> onSearch(searchValue)}>
```

Reply:
changed